### ISSUE TITLE
FEATURE REQ: Show `?` mark near things that are likely to confuse users and add a gif photo there

### ISSUE THREAD

**Issue by GaurangTandon** [GaurangTandon/ProKeys#122]
**Question:** Where do we need this, and isn't the help page enough?

(this issue will make more sense after the v3.3 update, when there's a burst of features)

**Comment by iWrote** [comment]
i was actually in favour of gifs until i realised they are super space heavy, gifs beat the help page any day

you could make gifs work by storing them on dropbox or something but then they wouldn't work offline

**Comment by GaurangTandon** [comment]
Yep, this issue is pretty much obsolete. I guess we could link the Help icon to a Youtube video instead. What say?

**Comment by iWrote** [comment]
Online only feature. Okay. Close.

**Comment by JAIDEV123** [comment]
Compromise, just add a tooltip on every '?'. I feel like chances are the Gif won't be clear anyway. So chances of it being truly effective are low but then if it was near a quarter screen, it would work well. Regardless, for user onboarding, we do need gif's right? In my head they have them at least, before the user gets to try the text editor out.

**Comment by GaurangTandon** [comment]
I do not know if we need gif's for user onboarding. I feel a single interactive textbox should do the job well.
